react/modules/gaEvents.ts

```diff
@@ -20,16 +22,16 @@ import {
 } from './utils'
 import shouldMergeUAEvents from './utils/shouldMergeUAEvents'
 
-export function viewItem(eventData: PixelMessage['data']) {
+export function viewItem(eventData: ProductViewData) {
   if (!shouldMergeUAEvents()) return
 
   const eventName = 'view_item'
 
-  const { currency, product, list, position } = eventData
```

Comment:
Typing `eventData` as `ProductViewData` helped uncover it doesn't contain `position` so it needed to be removed.

![CleanShot 2023-02-28 at 11 05 05](https://user-images.githubusercontent.com/381395/221877311-c09882f2-1baf-4e84-abc5-5d4feedd4e3f.png)